.cursor/rules/solidity-styles.mdc

```diff
@@ -0,0 +1,71 @@
+---
+description:
+globs:
+alwaysApply: true
```

Comment:
This will make sure it's only attached when in a solidity file:

```suggestion
---
description: Solidity Style Guide
globs: packages/contracts-bedrock/**/*.sol
alwaysApply: false
---
```